js/main.js

```diff
@@ -52,156 +54,51 @@ function goBackHandler(){
 
 $goBack.forEach(item => item.addEventListener('click', goBackHandler));
 
-//----------Render Push Exercises------------------
-
-function renderDelts() {
-  var numExercises = 0;
-  for (var i = Math.floor(Math.random() * exercises.delts.length); numExercises< 2; i++){
-    var $li = document.createElement('li');
-    $li.setAttribute('id', exercises.delts[i].uuid);
-    var $p = document.createElement('p');
-    $p.textContent = exercises.delts[i].name;
-    $li.appendChild($p)
-    $deltsList.appendChild($li);
-    numExercises++;
-      }
-    }      
-
-$pushButton.addEventListener('click', renderDelts);
-
-
-// ---------Render Chest -------------------
-
-function renderChest(){
-  var numExercises = 0;
-  for (var i = Math.floor(Math.random() * exercises.chest.length); numExercises< 2; i++){
-    var $li = document.createElement('li');
-    $li.setAttribute('id', exercises.chest[i].uuid);
-    var $p = document.createElement('p');
-    $p.textContent = exercises.chest[i].name;
-    $li.appendChild($p)
-    $chestList.appendChild($li);
-    numExercises++;
-      }
-}
-
-$pushButton.addEventListener('click', renderChest);
-
-// ----------Render Triceps-----------------
-
-function renderTriceps(){
-  var numExercises = 0;
-  for (var i = Math.floor(Math.random() * exercises.triceps.length); numExercises< 2; i++){
-    var $li = document.createElement('li');
-    $li.setAttribute('id', exercises.triceps[i].uuid);
-    var $p = document.createElement('p');
-    $p.textContent = exercises.triceps[i].name;
-    $li.appendChild($p)
-    $tricepsList.appendChild($li);
-    numExercises++;
-      }
-}
-$pushButton.addEventListener('click', renderTriceps)
+//----------Render Push Exercises------------------    
 
-//---------Render Quads ----------------
+function renderExercises(id){
+ for (var key in exercises[id]){
+   var eachExerciseGroup = exercises[id][key];
+   var randomIndexArray = [genRandomIndex(eachExerciseGroup), genRandomIndex(eachExerciseGroup)];
+   renderList(randomIndexArray, eachExerciseGroup, key);
+    }
+  }   
 
-function renderQuads(){
-  var numExercises = 0;
-  for (var i = Math.floor(Math.random() * exercises.quads.length); numExercises< 2; i++){
-    var $li = document.createElement('li');
-    $li.setAttribute('id', exercises.quads[i].uuid);
-    var $p = document.createElement('p');
-    $p.textContent = exercises.quads[i].name;
-    $li.appendChild($p)
-    $quadsList.appendChild($li);
-    numExercises++;
-      }
+function findTargetList(key){
+  for (var i = 0; i < $targetList.length; i++){
+    if ($targetList[i].getAttribute('data-list') === key){
```

Comment:
Instead of doing a loop, you should be able to just do `document.querySelector('[data-list="' + key + '"]')` 🤓